pkg/specgenutil/specgen.go

```diff
@@ -409,7 +409,7 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *entities.ContainerCreateOptions
 	s.WorkDir = c.Workdir
 	if c.Entrypoint != nil {
```

Comment:
Following block is only invoked when `--entrypoint` is set from CLI or `remote-client`